app/views/vm_common/_live_migrate.html.haml

```diff
@@ -1,68 +1,7 @@
-%form#form_div{:name           => "angularForm",
-               'ng-controller' => "vmCloudLiveMigrateFormController as vm",
-               "miq-form"      => true,
-               "model"         => "vm.vmCloudModel",
-               "model-copy"    => 'vm.modelCopy',
-               'form-changed'  => ''}
-  = render :partial => "layouts/flash_msg"
-  %h3
-    = _('Migrate Instance')
-  .form-horizontal
-    .form-group
-      %label.col-md-2.control-label
-        = _('Auto-select Host?')
-      .col-md-8
-        %input{:type         => "checkbox",
-               :name         => "auto_select_host",
-               :required     => true,
-               'ng-model'    => "vm.vmCloudModel.auto_select_host",
-               'ng-disabled' => "vm.hosts.length == 0"}
+- message = n_("1 Instance to be Live Migrated", "%{amount} Instances to be Live Migrated", @live_migrate_items.length) % {:amount => @live_migrate_items.length}
+- record_id = @live_migrate_items.length == 1 ? @live_migrate_items[0].id : ''
 
-        %input{:type         => "hidden",
-               :name         => "auto_select_host",
-               :value        => "on",
-               'ng-if'       => "vm.hosts.length == 0"}
-    .form-group
-      %label.col-md-2.control-label
-        = _('Block Migration')
-      .col-md-8
-        %input{:type        => "checkbox",
-               :name        => "block_migration",
-               :required    => true,
-               'ng-model'   => "vm.vmCloudModel.block_migration"}
-    .form-group
-      %label.col-md-2.control-label
-        = _('Disk Over Commit')
-      .col-md-8
-        %input{:type        => "checkbox",
-               :name        => "disk_over_commit",
-               'ng-model'   => "vm.vmCloudModel.disk_over_commit",
-               :required    => true}
-  #live-migrate-select-destination-host{'ng-hide' => 'vm.vmCloudModel.auto_select_host'}
-    %h3
-      = _('Select Destination Host')
-    .form-horizontal
-      .form-group
-        %label.col-md-2.control-label
-          = _('Destination Host')
-        .col-md-8
-          %select{:name      => 'destination_host_id',
-                  'ng-model' => 'vm.vmCloudModel.host',
-                  'ng-options' => 'host.name as host.name for host in vm.hosts track by host.id'}
-
-  %div_for_paging{'ng-controller'                    => "pagingDivButtonGroupController",
-                  'paging_div_buttons_state_enabled' => true,
-                  'paging_div_buttons_id'            => "angular_paging_div_buttons",
-                  'paging_div_buttons_type'          => "Submit"}
-
-  %hr
-  %div
-    %h3
-      = n_("1 Instance to be Live Migrated", "%{amount} Instances to be Live Migrated", @live_migrate_items.length) % {:amount => @live_migrate_items.length}
```

Comment:
Looks like the message belongs together with the GTL.
So we should probably keep them both in the original template.